week-2/G-conditionals-2/exercise-3.js

```diff
@@ -9,6 +9,7 @@
 */
 
 function calculateGrade(mark) {
+  return (mark >= 80)?"A":(mark >=60 && mark <80)?"B":(mark >= 50 && mark < 60)?"C":"F";
```

Comment:
I generally don't like chaining multiple ternary operators together. I think it makes it quite difficult to read and work out what it is doing. An `if` statement is much easier to read.

In addition I think you can improve the conditional here - because the first conditional checks that `mark` is greater than 80, then you don't need to check that again in the second conditional.